### PR TITLE
Improved Polish Localisation

### PR DESCRIPTION
**Pull request by HackZy01**
### 1.What's changed

- Fixed the top menu layout
- Fixed the grammar
- It's coherent and easier to understand

**Before:**</br><br>**After:**<br>

### CODE REVIEW

**Comment by HackZy01** [comment]
# 1.What's changed

- Fixed the top menu layout
- Fixed the grammar
- Used some idioms for fun

# 2. What's important in a translation 
Sticking to the original isn't important, but **getting the point across is**

**Comment by HackZy01** [comment]
Also wtf in google translate mixed with deepl was the original

File: texts/index.js [hunk 1]
```diff
@@ -301,47 +301,48 @@ texts.push([
 	"Nog niet", // 38(made by nishinji, translated by deepl)
 ])
 
-// Polish (Translation provided by circl, corrected by kacek)
+
```
Comment (nishinji) [inline texts/index.js:304]: delete

File: texts/index.js [hunk 2]
```diff
@@ -301,47 +301,48 @@ texts.push([
...
+	"Dołóż swoją cegięłke!", // 32
 	"Zgłoś problem lub zaproponuj zmiany w kodzie na naszym <a href=\"https://github.com/Vita3K/Vita3K\">Githubie</a>.", // 33
-	"Co nowego", // 34
-	"Prowadzimy bloga gdzie publikujemy irreguralnie szczegółowe raporty z najnowszym postępem. Przeczytaj co nowego!", // 35
+	"Sprawdź co nowego", // 34
+	"Prowadzimy bloga gdzie nieregularnie publikujemy szczegółowe raporty z najnowszym postępem. Serdecznie zachęcamy do przeczytania!", // 35
 	"Quickstart", // 36
 	"Blog", // 37
-	"Jeszcze nie", // 38(made by nishinji, translated by deepl)
+	"Jeszcze nie dostępne", // 38(made by nishinji, translated by deepl)
```
Comment (nishinji) [inline texts/index.js:345]: can you delete my comment?

**Comment by nishinji** [comment]
Thank you very much.
are there any translation corrections for compatibility page?

**Comment by HackZy01** [comment]
I can make them

**Comment by HackZy01** [comment]
Compatibility page done

**Review by Zangetsu38 — APPROVED** [review]

**Pull request merged**